packages/flutter/lib/src/material/slider.dart

```diff
@@ -460,14 +466,16 @@ class _RenderSlider extends RenderBox {
     markNeedsPaint();
   }
 
-  double get textScaleFactor => _textScaleFactor;
-  double _textScaleFactor;
+  MediaQueryData get mediaQueryData => _mediaQueryData;
+  MediaQueryData _mediaQueryData;
 
```

Comment:
extraneous blank line

Reply:
The rest of the file was like that, so I did the same.  Removed (from all setter/getter/var triplets).